src/main/java/org/spongepowered/api/util/mapitem/MapItem.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.util.mapitem;
+
+import java.awt.Color;
+import java.util.List;
+
+/**
+ * Represents a map.
+ */
+public interface MapItem {
+    
+    /**
+     * Gets this map's x center.
+     *
+     * @return The x center of the map.
+     */
+    Integer getXCenter();
+
+    /**
+     * Gets this map's z center.
+     *
+     * @return The z center of the map.
+     */
+    Integer getZCenter();
+
+    /**
+     * Gets this map's dimension.
+     *
+     * @return The dimension ID of the map.
+     */
+    Integer getDimensionId();
+
+    /**
+     * Gets this map's scale.
+     *
+     * @return The scale of the map.
+     */
+    Integer getScale();
+
+    /**
+     * Gets the visible players by name on this map.
+     *
+     * @return The list of players visible.
+     */
+    List<String> getVisiblePlayers();
+    
+    /**
+     * Gets the colors on this map.
+     *
+     * @return The list of colors.
+     */
+    List<Color> getColors();
```

Comment:
The map is 2-dimensional but this is a list (1-dimensional) I don't think this method really makes sense. Perhaps there should be `getColor(Vector2i)`